src/Components/Blazor/testassets/HostedInAspNet.Server/Program.cs

```diff
@@ -14,12 +13,12 @@ public static void Main(string[] args)
             BuildWebHost(args).Run();
         }
 
-        public static IWebHost BuildWebHost(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .UseConfiguration(new ConfigurationBuilder()
-                    .AddCommandLine(args)
-                    .Build())
-                .UseStartup<Startup>()
-                .Build();
+        public static IHost BuildWebHost(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+            .ConfigureWebHostDefaults(webHostBuilder =>
+            {
+                webHostBuilder.UseStartup<Startup>();
+            })
```

Comment:
@Tratcher - this is the state of the art right?

Reply:
Almost: https://github.com/aspnet/AspNetCore/blob/96f1c92caa484e6b9c7e5d3768c7dbe70287d87f/src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Program.cs#L14-L24